CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 Observes [Semantic Versioning](https://semver.org/spec/v2.0.0.html) standard and 
 [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) convention.
 
+## [0.2.0] - 2022-09-20
```

Comment:
I think adding a notebook is a minor bump `0.1.3`

Reply:
Updated, thanks